src/helpers/date-helpers.ts

```diff
@@ -189,3 +189,18 @@ export function getTimeOfDayString(dateOrDateString: Date | string) {
 	}
 	return formatDateForLocale(date, "p");
 }
+
+/** e.g., 12 P (localized, may be 24h) - a time of 00:00:00 is returned as an empty string */
+export function getShortTimeOfDayString(dateOrDateString: Date | string) {
+	const date = toDate(dateOrDateString);
+	if (!date || (date.getHours() === 0 && date.getMinutes() === 0 && date.getSeconds() === 0)) {
+		return "";
+	}
+	// date-fns doesn't have a format for this one, so we have to fall back to Intl
+	const locale = getIntlLocale();
```

Comment:
This is not working with the storybook language toggler. 
I did confirm it will respect my browser's language settings.